cli_tools/osconfig_agent/patch_window.go

```diff
@@ -274,7 +274,9 @@ func patchRunner() {
 			reboot := pw.run()
 			if reboot {
 				fmt.Println("DEBUG: reboot requested", pw.Name)
-				// TODO: Actually reboot.
+				if err := rebootSystem(); err != nil {
+					fmt.Println("ERROR: error running reboot:", err)
+				}
 				os.Exit(0)
```

Comment:
This only happens if we fail to reboot. Should we log the fact that we're exiting so it doesn't look like a crash?